docs/_toc.yml

```
@@ -84,6 +84,13 @@ subtrees:
       - file: plugins/debug_plugins
       - file: plugins/npe2_migration_guide
       - file: plugins/npe1
+      - file: plugins/testing_workshop_docs/index
+        subtrees:
+        - entries:
+          - file: plugins/testing_workshop_docs/Pythons-assert-keyword
+          - file: plugins/testing_workshop_docs/Pytest-testing-frameworks
+          - file: plugins/testing_workshop_docs/Readers-and-fixtures
+          - file: plugins/testing_workshop_docs/Test-coverage
```

I would recommend also adding the doc with the links to resources to the table of contents

```suggestion
          - file: plugins/testing_workshop_docs/Test-coverage
          - file: plugins/testing_workshop_docs/Testing-Resources
```